package.json

```diff
@@ -2,19 +2,44 @@
   "name": "@api3/chains",
   "license": "MIT",
   "version": "1.4.0",
+  "author": "API3 DAO",
   "private": false,
-  "main": "src/index",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/es6/index.js",
+  "repository": "git@github.com:api3dao/chains",
+  "bugs": {
+    "url": "https://github.com/api3dao/chains/issues"
+  },
   "files": [
-    "chains",
+    "dist",
     "src"
   ],
   "scripts": {
-    "check-provider": "node scripts/check-provider",
+    "build:cjs": "yarn tsc -p tsconfig.build-cjs.json",
+    "build:es6": "yarn tsc -p tsconfig.build-es6.json",
+    "build:esm": "yarn tsc -p tsconfig.build-esm.json",
+    "build": "yarn clean && yarn generate:chains && yarn build:cjs && yarn build:es6 && yarn build:esm",
+    "clean": "rimraf ./dist",
+    "dev": "ts-node scripts/combine-chains.ts --watch",
+    "chains:rename": "ts-node scripts/rename-chain-file.tss",
+    "env:write": "ts-node scripts/write-env-variables.ts",
+    "generate:chains": "ts-node scripts/generate-chains.ts",
+    "prepublishOnly": "yarn validate:chains",
```

Comment:
Why validate only chains?

I think you want to define `validate` task that runs all of the validations (I'd also run that on CI instead of running the validations one-by-one).